doc/usage.md

```diff
@@ -16,16 +16,16 @@ OPTIONS:
                                         easily (e.g. test and production 
                                         setup).Sample of configuration file 
                                         parameters looks like:
-                                        vss = vss_release_2.2.json
+                                        vss = vss_release_2.1.json
                                         cert-path = . 
                                         log-level = ALL
                                         
   --vss arg                             [mandatory] Path to VSS data file 
                                         describing VSS data tree structure 
                                         which `kuksa-val-server` shall handle. 
-                                        Sample 'vss_release_2.2json' file can 
+                                        Sample 'vss_release_2.1.json' file can 
```

Comment:
While I understand that this is just an example, any particular reason of changing the v2.2 to v2.1?

Reply:
Ah! The root cause was, I updated usage.md by copying the output form `kuksa-val-server --help`, it seems that was never updated, probably the doc was "manually" edited when upgrading to 2.2 and we forgot the actual compiled in usage notes. Should be fixed now